packages/fuels-contract/src/contract.rs

```diff
@@ -160,8 +168,25 @@ impl Contract {
         binary_filepath: &str,
         wallet: &LocalWallet,
         params: TxParameters,
+        storage_configuration: StorageConfiguration,
```

Comment:
in #411 it was discussed that `deploy` should stay as simple as possible. If the user wants advanced features like salt and storage they should use `deploy_with_parameters`. Perhaps we should move this to `deploy_with_parameters`.

Reply:
You're right, maybe we should. I'll move it!